src/core/analytics.js

```diff
@@ -352,7 +352,7 @@ class Analytics {
               self.failedToBeLoadedIntegration.push(intgInstance);
             }
           }
-        }, 100);
+        }, 1);
```

Comment:
this is too intensive I belive in terms of cpu consumeption imagine 8 destinations that take 300ms to load. how may times will you invoke this method?